lua/swenv/api.lua

```diff
@@ -6,13 +6,34 @@ local ORIGINAL_PATH = vim.fn.getenv('PATH')
 
 local current_venv = nil
 
+local get_conda_base = function(conda_exe_path)
+  local parts = {}
+
+  for part in conda_exe_path:gmatch("[^/]+") do
+    table.insert(parts, part)
+  end
+  table.remove(parts) -- remove the last part
+  table.remove(parts) -- remove the second-to-last part
+  return "/" .. table.concat(parts, "/")
+end
+
 local set_venv = function(venv)
-  current_venv = venv
-  local venv_bin_path = venv.path .. '/bin'
-  vim.fn.setenv('PATH', venv_bin_path .. ':' .. ORIGINAL_PATH)
-  vim.fn.setenv('VIRTUAL_ENV', venv.path)
-  if settings.post_set_venv then
-    settings.post_set_venv(venv)
+  if venv.source == "conda" then
+    vim.fn.setenv('CONDA_PREFIX', venv.path)
+    vim.fn.setenv('CONDA_DEFAULT_ENV', venv.name)
+    vim.fn.setenv('CONDA_PROMPT_MODIFIER', "(" .. venv.name .. ")")
+    vim.cmd("LspRestart")
```

Comment:
Maybe this is indeed good to do but I wasn't sure if it had undesired consequences. Anyway, I think we should do it both for conda and venv or neither. Or would there be a reason to do it only for conda?

Reply:
I think it's necessary for the changes to take effect.